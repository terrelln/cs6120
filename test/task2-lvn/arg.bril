# ARGS: 3 5
@main(a: int, b: int) {
    c: int = id a;
    c: int = add a b;
    c: int = add a b;
    c: int = id a;
    c: int = id a;
    c: int = id b;
    c: int = id c;
    c: int = add a b;
    c: int = mul a b;
    c: int = mul a b;
    c: int = add a b;
    c: int = mul a b;
    d: int = add a b;
    e: int = add a b;
    f: int = add d e;
    g: int = add a b;
    h: int = add a b;
    i: int = add g h;
    print c;
    print f;
    print i;
}