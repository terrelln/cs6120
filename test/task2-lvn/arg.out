@main(a: int, b: int) {
.__block0:
  __var2: int = add a b;
  f: int = add __var2 __var2;
  print f;
}
